doc/source/development/contributing_environment.rst

```diff
@@ -87,14 +88,24 @@ Option 1: using mamba (recommended)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 * Install `mamba <https://mamba.readthedocs.io/en/latest/installation/mamba-installation.html>`_
+  (the instructions may direct you to install `miniforge <https://github.com/conda-forge/miniforge>`_,
+  which is OK - it includes mamba)
```

Comment:
```suggestion
* Install miniforge to get `mamba <https://mamba.readthedocs.io/en/latest/installation/mamba-installation.html>`_
```

Reply:
Thank you - great suggestion

Reply:
With this suggestion is now looks like this.  Confirmed link works.

<kbd><img width="783" alt="image" src="https://github.com/pandas-dev/pandas/assets/122238526/bb3c8dc6-ba1b-427b-b39d-f5b8cc460d98"></kbd>